### PR TITLE
Enable the workload certificate API

### PR DESCRIPTION
**Pull request by lei-tang**
Enable the workload certificate API.

### CODE REVIEW

File: asmcli/lib/installation_dependencies.sh [hunk 1]
```diff
@@ -186,6 +186,12 @@ enable_gcloud_apis(){
   # shellcheck disable=SC2046
   retry 3 gcloud services enable --project="${PROJECT_ID}" $(required_apis | tr '\n' ' ')
 
+  local CA; CA="$(context_get-option "CA")"
+  if [[ "${CA}" = "managed_cas" ]]; then
+    x_enable_gke_hub_api "autopush-gkehub.sandbox.googleapis.com"
```
Comment (zerobfd) [inline asmcli/lib/installation_dependencies.sh:191]: If the dependencies aren't in prod yet then we generally don't push to the main development branch. Is there a strong reason that we need it in GitHub at this level of development?

**Review by zerobfd — CHANGES_REQUESTED** [review]
If you look at the "required_apis()" function there's already a place where we determine apis based off of dependencies. We should put the logic there.

File: asmcli/lib/installation_dependencies.sh (hunk 1 above)
Comment (lei-tang) [reply asmcli/lib/installation_dependencies.sh:191]: The dependencies are not in prod yet. I agree with you on the the comments. I will update this PR when the dependencies are in prod. After the PR is updated, I will ask for your review again. Thanks!

Comment (lei-tang) [reply asmcli/lib/installation_dependencies.sh:191]: The dependencies are not in prod yet. I agree with you on the the comments. I will update this PR when the dependencies are in prod. After the PR is updated, I will ask for your review again. Thanks!

Comment (lei-tang) [reply asmcli/lib/installation_dependencies.sh]: @zerobfd I have updated the PR based on your comment. Please take another look. Thanks!

**Review by zerobfd — APPROVED** [review]

**Pull request merged**